### DISCUSSION TITLE
Customize Filter  Query for Filament Admin List

### DISCUSSION THREAD

**Discussion by MPTA** [filamentphp/filament#3168] [Help]
hi, i have 2 lists : User & UserTransactions (user_id is foreign key in UserTransactions table ) , one user has many transactions , i calculate user credit by sum of user transactions records. i want to create a new filter to show only users who credit is > 0 . would you please let me know how i can define by query builder ?

**Comment by danharrin** [comment]
Have you worked out how to do this in Eloquent? Once you've done that, you can just pass it to the `query()` method of the Filter
